### PR TITLE
PLP

### PR DESCRIPTION
**Pull request by mango-db**
### Stories/Links:

[DOP-2094](https://jira.mongodb.org/browse/DOP-2094).
[Docs-compass PR](https://github.com/mongodb/docs-compass/pull/342).
[Figma](https://www.figma.com/file/BJLheYUTCPkY7KoiaumVHA/PLP-Template).
[PD](https://docs.google.com/document/d/1sjg8Hagdbt02qQa7ir3wWU6-JAi1X3bt4OeF52yUpvI/edit#).
[Ye ol tech spec](https://docs.google.com/document/d/13nl7VosWePqKCSU3liQ0uZnMvmM1KWMcn4aYaHnSYfM/edit).

### Staging Links:

[Docs-compass](https://docs-mongodbcom-staging.corp.mongodb.com/DOP-1979/compass/cgoecknerwald/plp/).
[Docs-landing](https://docs-mongodbcom-staging.corp.mongodb.com/master/landing/cgoecknerwald/plp/). <- staging to confirm that nothing funky is happening with landing
[Docs-datalake](https://docs-mongodbcom-staging.corp.mongodb.com/master/datalake/cgoecknerwald/plp/) <- additional anti-funky sanity check

### .env

```
GATSBY_SITE=compass
GATSBY_PARSER_USER=cgoecknerwald
GATSBY_PARSER_BRANCH=DOP-1979
GATSBY_SNOOTY_DEV=true
```

### Notes:

- Sidebar to be handled by docs nav redesign

### CODE REVIEW

**Review by allisonmui9 — APPROVED** [review]
Looks great! LGTM ty Claire :) !!

**Comment by sophstad** [comment]
@mango-db Compass staging is looking a bit funky for me. Maybe restaging or purging Fastly would help?
image

**Comment by mango-db** [comment]
Restaged all 3!

**Comment by mango-db** [comment]
Staging has been updated to reflect changes (rebase from master)

File: src/components/landing/Card.js [hunk 1]
```diff
@@ -60,16 +89,24 @@ const Card = ({
   },
 }) => {
   const Card = isCompact || isExtraCompact ? CompactCard : StyledCard;
+  const Icon = isCompact ? CompactIcon : CardIcon;
   return (
     <Card
       onClick={() => {
         window.location.href = url;
       }}
     >
```
Comment (casthewiz) [inline src/components/landing/Card.js:98]: I think we discussed some safeguards around nullable and divergent `url` values from potential urls in child components, and handling it in the onClick with nullable logic. 

I don't think handling it should block release, but if we don't add handling in this changeset then we should flag it as a fast follow-up TODO item.

File: src/components/Tabs.js [hunk 2]
```diff
@@ -20,17 +20,55 @@ const getPosition = (element) => {
...
+    }
+
+    @media ${theme.screenSize.upToSmall} {
+      margin-bottom: 40px;
+    }
+  }
 `;
 
-const Tabs = ({ nodeData: { children, options = {} }, ...rest }) => {
+const getTabsStyling = ({ isHidden, isProductLanding }) => css`
```
Comment (mango-db) [inline src/components/Tabs.js:45]: Context for @casthewiz: `LeafyTab` was behaving weirdly with unexpected props, so this is the workaround.

**Review by casthewiz — APPROVED** [review]

**Review by rayangler — APPROVED** [review]
Great work!

**Pull request merged**